Assets/Unity3DTiles/Unity3DTilesetTraversal.cs

```diff
@@ -207,22 +219,22 @@ void MarkUsedRecursively(Unity3DTile tile)
         void MarkUsedSetLeaves(Unity3DTile tile)
         {
             // A used leaf is a node that is used but has no children in the used set
-            if(!tile.FrameState.IsUsedThisFrame(this.frameCount))
+            if (!tile.FrameState.IsUsedThisFrame(this.frameCount))
             {
                 // Not used this frame, can't be a used leaf and neither can anything beneath us
                 return;
             }
             this.tileset.Statistics.UsedSetCount += 1;
             // If any child is used, then we are not a leaf
             bool anyChildrenUsed = false;
-            for(int i = 0; i < tile.Children.Count; i++)
+            for (int i = 0; i < tile.Children.Count; i++)
             {
                 anyChildrenUsed = anyChildrenUsed || tile.Children[i].FrameState.IsUsedThisFrame(this.frameCount);
             }
-            if(!anyChildrenUsed)
+            if (!anyChildrenUsed || ForceTiles.Contains(tile))
```

Comment:
Is it a problem if the search terminates before reaching a tile in ForceTiles?

Reply:
hm, that might have been possible, yes, I pushed a fix